src/fu-security-attr.c

```diff
@@ -203,6 +203,598 @@ fu_security_attr_get_name(FwupdSecurityAttr *attr)
+		if (fwupd_security_attr_has_flag(attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS)) {
+			/* TRANSLATORS: longer success message */
+			g_ptr_array_add(
+			    strs,
+			    _("Your system startup is being protected by Secure Boot."));
+		} else {
+			g_ptr_array_add(strs,
+					/* TRANSLATORS: longer failure message */
+					_("System startup is not being protected by Secure Boot "
+					  "which would allow an attacker to install software run "
+					  "before your operating system starts."));
+			check_bios_settings = TRUE;
+		}
+	}
+
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_TPM_EMPTY_PCR) == 0) {
+		if (fwupd_security_attr_has_flag(attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS)) {
+			/* TRANSLATORS: longer success message (unlikely to see) */
+			g_ptr_array_add(strs, _("System attestation values have been set."));
+		} else {
+			g_ptr_array_add(strs,
+					/* TRANSLATORS: longer failure message */
+					_("System attestation values have not been set and "
+					  "firmware protection may be incomplete."));
+			contact_hw_vendor = TRUE;
+		}
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_TPM_RECONSTRUCTION_PCR0) == 0) {
+		if (fwupd_security_attr_has_flag(attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS)) {
+			g_ptr_array_add(
+			    strs,
+			    /* TRANSLATORS: longer success message (unlikely to see) */
+			    _("System firmware and startup measurements have been verified."));
+		} else {
+			g_ptr_array_add(
+			    strs,
+			    /* TRANSLATORS: longer failure message */
+			    _("System firmware and startup measurements could not be verified and "
+			      "so firmware protection may be incomplete."));
+			contact_hw_vendor = TRUE;
+		}
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_TPM_VERSION_20) == 0) {
+		if (fwupd_security_attr_has_flag(attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS)) {
+			g_ptr_array_add(strs,
+					/* TRANSLATORS: longer success message (unlikely to see) */
+					_("System firmware and startup measurements are being "
+					  "collected for verification."));
+		} else {
+			g_ptr_array_add(strs,
+					/* TRANSLATORS: longer failure message */
+					_("System firmware and startup measurements cannot be "
+					  "being collected for verification."));
+		}
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_MEI_MANUFACTURING_MODE) == 0 ||
+	    g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_PLATFORM_FUSED) == 0) {
+		if (fwupd_security_attr_has_flag(attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS)) {
+			g_ptr_array_add(strs,
+					/* TRANSLATORS: longer success message (unlikely to see) */
+					_("The system has been correctly configured when assembed "
+					  "by the vendor and firmware is being protected."));
+		} else {
+			g_ptr_array_add(
+			    strs,
+			    /* TRANSLATORS: longer failure message */
+			    _("The system has not been correctly configured when assembed by the "
+			      "vendor, which may allow an attacker to replace system firmware."));
```

Comment:
maybe replace vendor with the name of the vendor

Reply:
I guess we could do `your OS` -> `Fedora` too, right?

Reply:
Yes

Reply:
Hmm. We can't do this unless we do the $OS$ substitution client side; otherwise the translation IDs don't match and `dgettext("fwupd",str)` doesn't work. I suppose we could export `fwupd_client_get_host_vendor` for this and maybe even `fwupd_client_get_os_vendor()` but it's getting a bit unwieldy then.

Reply:
Ahh; maybe we can use `fwupd_security_attr_add_metadata()` -- will ponder.

Reply:
I think switching out host vendor is more important than OS vendor is FWIW.  I think there is less that you'll contact Red Hat or Canonical about than you would a HW vendor